src/main/java/pl/agh/edu/iisg/io/vmms/vmmsbackend/service/UserServiceImpl.java

```diff
@@ -42,7 +42,7 @@ public User save(User user) {
     @Override
     public User save(String name) {
         User user = new User();
-        user.setUserName(name);
+        //user.setUserName(name);
```

Comment:
I guess this shouldn't be commented? If this caused You compilation errors, try enabling annotation processing for Lombok

Reply:
thanks!